core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -652,7 +652,10 @@ private StorageDirView allocateSpace(long sessionId, AllocateOptions options) {
           // Block expansion are forcing the location. We do not want the review's opinion.
           dirView = mAllocator.allocateBlockWithView(sessionId, options.getSize(),
               options.getLocation(), allocatorView.refreshView(), true);
-          LOG.debug("Allocation after freeing space for block expansion: {}", dirView);
+          if (LOG.isDebugEnabled()) {
+            LOG.debug("Allocation after freeing space for block expansion: {}",
```

Comment:
This should probably be something like "Allocation after freeing space for block expansion. availableBytes: {}". Is the goal of this message to show the available bytes after freeing?